### ISSUE TITLE
Rapidly Clearing Player Model Throws Error

### ISSUE THREAD

**Issue by kalkronline** [TheOneTrueZandra/Figura#99]
```
[18:48:28] [ForkJoinPool.commonPool-worker-37/INFO] (Minecraft) [STDERR]: java.lang.NullPointerException: Cannot read field "playerId" because "net.blancworks.figura.PlayerDataManager.localPlayer" is null
[18:48:28] [ForkJoinPool.commonPool-worker-37/INFO] (Minecraft) [STDERR]: 	at net.blancworks.figura.PlayerDataManager.clearLocalPlayer(PlayerDataManager.java:157)
[18:48:28] [ForkJoinPool.commonPool-worker-37/INFO] (Minecraft) [STDERR]: 	at net.blancworks.figura.network.NewFiguraNetworkManager.lambda$deleteAvatar$3(NewFiguraNetworkManager.java:173)
[18:48:28] [ForkJoinPool.commonPool-worker-37/INFO] (Minecraft) [STDERR]: 	at java.base/java.util.concurrent.CompletableFuture$UniRun.tryFire(CompletableFuture.java:783)
[18:48:28] [ForkJoinPool.commonPool-worker-37/INFO] (Minecraft) [STDERR]: 	at java.base/java.util.concurrent.CompletableFuture.postComplete(CompletableFuture.java:506)
[18:48:28] [ForkJoinPool.commonPool-worker-37/INFO] (Minecraft) [STDERR]: 	at java.base/java.util.concurrent.CompletableFuture$AsyncRun.run(CompletableFuture.java:1806)
[18:48:28] [ForkJoinPool.commonPool-worker-37/INFO] (Minecraft) [STDERR]: 	at java.base/java.util.concurrent.CompletableFuture$AsyncRun.exec(CompletableFuture.java:1792)
[18:48:28] [ForkJoinPool.commonPool-worker-37/INFO] (Minecraft) [STDERR]: 	at java.base/java.util.concurrent.ForkJoinTask.doExec(ForkJoinTask.java:290)
[18:48:28] [ForkJoinPool.commonPool-worker-37/INFO] (Minecraft) [STDERR]: 	at java.base/java.util.concurrent.ForkJoinPool$WorkQueue.topLevelExec(ForkJoinPool.java:1016)
[18:48:28] [ForkJoinPool.commonPool-worker-37/INFO] (Minecraft) [STDERR]: 	at java.base/java.util.concurrent.ForkJoinPool.scan(ForkJoinPool.java:1665)
[18:48:28] [ForkJoinPool.commonPool-worker-37/INFO] (Minecraft) [STDERR]: 	at java.base/java.util.concurrent.ForkJoinPool.runWorker(ForkJoinPool.java:1598)
[18:48:28] [ForkJoinPool.commonPool-worker-37/INFO] (Minecraft) [STDERR]: 	at java.base/java.util.concurrent.ForkJoinWorkerThread.run(ForkJoinWorkerThread.java:183)
```
Caused by rapidly clicking the big red X found in the model menu.

**Comment by TheOneTrueZandra** [comment]
Yeah this one makes sense, thanks for the report! I’ll look into fixing it later.

**Comment by kalkronline** [comment]
fixed by pr#100
